pkg/workload/ycsb/ycsb.go

```diff
@@ -937,6 +962,10 @@ func (yw *ycsbWorker) chooseOp() operation {
 		return insertOp
 	}
 	p -= yw.config.insertFreq
+	if readOnly.Load() == 0 && p <= yw.config.deleteFreq {
```

Comment:
It doesn't look like this `readOnly` global is hooked up to anything. It dates back at least as far as 08194fb4. Mind removing in a new commit?